### ISSUE TITLE
Link to Video showing how Treat files as binary is ACCIDENTALLY Clicked! (Plus TLDW; version)

### ISSUE THREAD

**Issue by kirkw** [stefankueng/grepWin#334]
Stefan,
  Text version.  The Treat file as binary checkbox is SIZED to the END of the REPLACE WITH box.  OUCH.
So, if I cover the window with my editor, then click below that box (to avoid any controls).  You turn this on, and goof your searches!!!

  Thanks for this tool (Combined with NimbleText for some processing), I was able to upgrade hundreds of files to call a routine differently, and then take the filenames in, and push them through to the database, recreating all of the code!!!  Every day I am using this great tool!

Here is the link to the video but it should be easy enough to validate.  [That checkbox should end at the TEXT "binary", IMO
https://www.screencast.com/t/YSisbihgzY

**Comment by stefankueng** [comment]
thanks for the video

**Comment by kirkw** [comment]
Stefan...   This is still happening in Windows 64 bit, ENGLISH...
How was it "fixed", the checkbox is still clickable all the way to the end of the REPLACE text box on a wide screen...

**Comment by hpwamr** [comment]
> How was it "fixed", the checkbox is still clickable all the way to the end of the REPLACE text box on a wide screen...

I think this PR has definitely fixed this issue: https://github.com/rizonesoft/Notepad3/pull/4223
Also in adjusting the length of "Treat files as binary" consider the length of the longest text (Greek language). 🤔 

![2022-11-17_001809](https://user-images.githubusercontent.com/34211772/202316000-7061cbf7-06f6-417f-9b74-6ca10ae046aa.png)

**Comment by kirkw** [comment]
Okay, here is a new video.  This is 3480x2160 full width...  I show it, apologies if you have to scroll really far to the right. LOL.
Just proves the point.  I doubt the GREEK language is that long, and it is SIZING with the dialog width, because PAST the text box above it, it will not click...  So, it is NOT going to the extreme right of the Form.

I hope this helps.  And I downloaded and re-installed today (well, did a Repair since I already had the correct version, just to confirm)

EDIT: PLEASE Excuse the SNARKINESS in my voice.  Wow...  I almost re-recorded this.  NOT the intention.
Zooming in on the video helps...
https://app.screencast.com/i8XgZ9tdxEJWK

**Comment by Dean-NC** [comment]
It's not fixed in the latest .15.
I ran into this problem with 2.0.10 (or 11?) when I couldn't figure out why searches were failing when I could clearly see the text in the file. I finally saw I had the binary checkbox turned on, which I knew I hadn't enabled. I uninstalled and then installed .15.
The binary checkbox label stretches almost to the end, whereas the checkbox below that ("keep file date...") is OK and is normal width.

I love the tool though, and have used it for many years as my only search tool. It's in the top list of things I install on any computer I use.

![grepWinBinaryCheck](https://github.com/stefankueng/grepWin/assets/12361341/367eef62-7ce2-4701-b91a-891c3a7ececa)
